### ISSUE TITLE
Loop in the reviewpad action

### ISSUE THREAD

**Issue by marcelosousa** [reviewpad/reviewpad#303]
**Describe the bug**
It seems that adding the Show model to a pull request, e.g. https://github.com/reviewpad/reviewpad/pull/302 induces the action into a loop!

**Comment by zolamk** [comment]
After investigating what causes the loop, I've discovered the following things to be causing the loop
- creating a PR with the `show` option doesn't cause the loop
- creating a PR with the `ask` option doesn't cause the loop
- creating a PR with the `ask` option and then switching it to the `show` option causes the loop due to the following reasons
when switching from the `ask` option to the `show` option, two pull request edited events will be triggered, one when toggling `ask` off and a second one when toggling `show` to on.

The loop happens because if no model is selected, the current Reviewpad configuration assumes it's an ask and will label the PR `ask`, but because toggling `show` on also triggered a second event which determines the PR to be a `show` model will remove the `ask` label and add the `show` label. 

After which those actions removing and add labels will trigger a pull request labeled event, then the above action will be performed again and again in a loop.

A reproduction repo has been set up at https://github.com/zolamk/reviewpad-loop-playground.

In conclusion, the issue is not with Reviewpad it self, but the way the GitHub workflow actions are set up.

**Comment by marcelosousa** [comment]
Make sense - thanks for this investigation!
